core/src/dbs/session.rs

```diff
@@ -69,6 +72,17 @@ impl Session {
 		self.rt
 	}
 
+	/// Checks if the session has expired
+	pub(crate) fn expired(&self) -> bool {
+		match self.exp {
+			Some(exp) => Utc::now().timestamp() > exp,
+			// Sessions for system users using basic authentication have no expiration.
+			// TODO(gguillemas): Change this to "true" once session lifetime is configurable
```

Comment:
Well, custom tokens may not have an expiration

Reply:
It is true and would be allowed by the [JWT RFC](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4). I will change the comment considering this option. Thanks Micha!

Reply:
Resolved in [5ba15cd](https://github.com/surrealdb/surrealdb/pull/3561/commits/5ba15cd18b29a133304600b4c8caed66a1610317).